### ISSUE TITLE
API usage is ready

### ISSUE THREAD

**Issue by i0natan** [i0natan/node-chaos-monkey#3]
@TITAN9389 I pushed a code that serves all API routes with mock data and updated the documentation readme.md

On the repo homepage, please read only the part that starts with: "API usage".

Basically, you can just perform 'npm start' and approach the routes.

Is the documentation detailed enough?

**Comment by TITAN9389** [comment]
@i0natan Yes its all good thanks.

**Comment by TITAN9389** [comment]
@i0natan there is the CORS error , not allowing my localhost:3000 , i don't want to change your node code if you can do it later i'm using Chrome extension now to pass it.

**Comment by i0natan** [comment]
@TITAN9389 OK, adding now

**Comment by i0natan** [comment]
@TITAN9389 I added not only CORS but also serving static files from the "UI" folder.

So you can work on a different port, but also try to locate your files within the UI folder, build for production, and try serving from the server port. This would be the real world scenario so I suggest testing it. Kindly pull latest and LMK wether works for you?

Any estimate when you might be able to share a draft?

**Comment by TITAN9389** [comment]
yes okay Thanks ,i'll see and do the Build files as well , I'll share this later today @i0natan

**Comment by TITAN9389** [comment]
@i0natan i've connected to web socket ,but i'm getting Null data sometimes is that normal or what ?
![image](https://user-images.githubusercontent.com/27279495/47170244-9d80b800-d316-11e8-9b3c-94a2bea93781.png)

**Comment by TITAN9389** [comment]
@i0natan as i knew the NULL throwing error while rendering the UI , so i catch it now with error so just UI will ignore and just render the data. and the data keeps emitting is it like that in normal ? i mean if i let it open it will still send me data forever 9999...we can have a call tomorrow to explain everything better.
![image](https://user-images.githubusercontent.com/27279495/47173872-4da6ee80-d320-11e8-8d02-753fa3a65846.png)

**Comment by i0natan** [comment]
@TITAN9389 Currently there's no real data, it's a mock that throw infinite events, that won't be that way in real-scenario - there won't be more than 10 items. Just ignore it.

Any chance you can share some prtscn, video clip or push code? just want to get a sense of how the UI looks like now

**Comment by TITAN9389** [comment]
i pushed the code to my Branch , merge it and run npm start from ui folder ( didn't make build yet ).

**Comment by TITAN9389** [comment]
@i0natan @amaliaman  made some changes in the code (webapp.js) file now just run npm run build from UI folder then just npm start from main folder it will serve the build files.
